From: Michael S. Tsirkin <mst@redhat.com>
Subject: [PATCH] vhost: remove unused include
References: FATE#311977
Git-commit: bf5e0bd27f7cbaca4d52ae395bbf3715775efebd
Patch-mainline: v2.6.37

vhost.c does not need to know about sockets,
don't include sock.h

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Acked-by: Bruce Rogers <brogers@suse.com>
---
 drivers/vhost/vhost.c |    2 --
 1 file changed, 2 deletions(-)

Index: b/drivers/vhost/vhost.c
===================================================================
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -30,8 +30,6 @@
 #include <linux/if_packet.h>
 #include <linux/if_arp.h>
 
-#include <net/sock.h>
-
 #include "vhost.h"
 
 enum {
